Lib/ensurepip/__init__.py

```diff
@@ -11,15 +11,12 @@
 
 
 __all__ = ["version", "bootstrap"]
-
-
-_SETUPTOOLS_VERSION = "49.2.1"
-
-_PIP_VERSION = "20.2.3"
-
+_PACKAGE_NAMES = ('setuptools', 'pip')
```

Comment:
Why is this being set

Reply:
The PR didn't backport cleanly, and I had to manually fix the merge conflict. I didn't notice that this variable isn't used in the older version of the code. It's harmless, but can be fixed. It's *not* the cause of the test failures, sadly.